libkbfs/md_journal_test.go

```diff
@@ -162,6 +170,36 @@ func checkIBRMDRange(t *testing.T, uid keybase1.UID,
 	}
 }
 
+func BenchmarkMDJournalBasicMDv2(b *testing.B) {
+	_, _, id, signer, ekg, bsplit, tempdir, j :=
+		setupMDJournalTest(b)
+	defer teardownMDJournalTest(b, tempdir)
+
+	mdCount := 10
+	revision := MetadataRevision(10)
+	prevRoot := fakeMdID(1)
+	for i := 0; i < b.N; i++ {
```

Comment:
Yes; thanks!

Weirdly, after adding these, v3 is now slower. I've also changed `mdCount` to 500 and it's still marginally slower